packages/publish.js

```diff
@@ -67,11 +67,11 @@ const currentCwd = process.cwd();
 process.chdir(path.resolve(__dirname, '..'));
 
 function publishCommand(workspace) {
-  execSync(`yarn workspace ${workspace} publish --new-version ${version} --tag ${tag}`, {
```

Comment:
These three packages all need to be published - and the build is failing completely because of the hard coded version number here that doesn't match up to the version that has actually been generated for each of the packages.